justfile

```
@@ -67,6 +63,22 @@ check_v2 *FLAGS:
     cargo clippy {{ check_flags }} --features "${FEATURES}"  {{ FLAGS }}
     set +x
 
+check *FLAGS:
+    #! /usr/bin/env bash
+    set -euo pipefail
+
+    FEATURES="$(cargo metadata --all-features --format-version 1 | \
+        jq -r '
+            [ ( .workspace_members | sort ) as $package_ids # Store workspace crate package IDs in `package_ids` array
+            | .packages[] | select( IN(.id; $package_ids[]) ) | .features | keys[] ] | unique # Select all unique features from all workspace crates
+            | del( .[] | select( any( . ; . == ("v2", "merchant_account_v2", "payment_v2") ) ) ) # Exclude some features from features list
```

```suggestion
            | del( .[] | select( any( . ; test("(([a-z_]+)_)?v2") ) ) ) # Exclude some features from features list
```

You can use something like this for filtering out features based on regex, for this and the other similar recipes. This would exclude either `v2` or anything that matches `([a-z_]+)_v2` from the feature list.